### DISCUSSION TITLE
[Feature request] Anychest, allowing access to blocked.

### DISCUSSION THREAD

**Discussion by 456dev** [Zrips/CMI#1615] [Feature Requests]
**Description of feature request:**
A toggleable command like silent chest, allowing access to chests with blocks above them and Shulker boxes with blocks in the way of them.

It could just not play the animation for shulkerboxes but still play the sound if silent chest is disabled

Command usage and example:
/cmi anychest (Playername) (-s)

/Cmi anychest on the456gamer

**Comment by TheBlueKingLP** [comment]
just a workaround for now, if silent chest is on, you can open any chest

**Comment by 456dev** [comment]
Thanks, it would just be nice to allow some VIP players to use any chest, and not silent chest
